ports/stm32/boards/WEACT_STM32H743/mpconfigboard.h

```diff
@@ -0,0 +1,135 @@
+/*
+ * The MIT License (MIT)
+ * Copyright (c) 2023 Matt Trentini
+ */
+
+#define MICROPY_HW_BOARD_NAME "WEACT_STM32H743"
+#define MICROPY_HW_MCU_NAME "STM32H743VIT6"
+#define MICROPY_HW_FLASH_FS_LABEL "WEACT_STM32H743"
+
+#define MICROPY_FATFS_EXFAT (1)
+#define MICROPY_HW_ENABLE_RTC (1)
+#define MICROPY_HW_ENABLE_RNG (1)
+#define MICROPY_HW_ENABLE_ADC (1)
+#define MICROPY_HW_ENABLE_DAC (1)
+#define MICROPY_HW_ENABLE_USB (1)
+#define MICROPY_HW_HAS_SWITCH (1)
+#define MICROPY_HW_HAS_FLASH (1)
+#define MICROPY_HW_ENABLE_SERVO (1)
+#define MICROPY_HW_ENABLE_TIMER (1)
+#define MICROPY_HW_ENABLE_SDCARD (1)
+#define MICROPY_HW_ENABLE_MMCARD (0)
+
+// Flash storage config
+#define MICROPY_HW_SPIFLASH_ENABLE_CACHE (1)
+#define MICROPY_HW_ENABLE_INTERNAL_FLASH_STORAGE (0)
+
+// Clock config
+#define MICROPY_HW_CLK_PLLM (5)
```

Comment:
Please add a comment stating the frequency of the external HSE crystal (and if it's a crystal or an oscillator).